tests/main/interfaces-custom-device-app-slot/task.yaml

```diff
@@ -28,6 +28,8 @@ prepare: |
     snap debug can-manage-refreshes | MATCH false
 
     snap ack "$TESTSLIB/assertions/testrootorg-store.account-key"
+    snap ack "$TESTSLIB/assertions/developer1.account"
```

Comment:
Will this ack work when we are running non-test keys build (aka a vanilla build)?